UnityProject/Assets/Scripts/ScriptableObjects/SpeechModifier.cs

```diff
@@ -76,7 +130,18 @@ string AddText(string message)
 			return message;
 		}
 
-		bool WasYelling(string message) => message == message.ToUpper();
+		bool WasYelling(string message)
```

Comment:
is this cheaper than ToUpper?

Reply:
It's a tiny bit faster. I think it's nice because it doesn't need to allocate another string.